File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/org/openapitools/model/PrivateUserData.kt
package org.openapitools.model

import java.util.Objects
import org.openapitools.model.GameProfileProperty
import org.openapitools.model.ReducedUserData
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 */
data class PrivateUserData (

        @JsonProperty("id") override val id: java.io.File? = null,

        @JsonProperty("properties") override val properties: List<GameProfileProperty>? = null
) : ReducedUserData{

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/org/openapitools/client/model/PrivateUserData.scala

package org.openapitools.client.model

import java.io.File

case class PrivateUserData (
    _id: Option[File],
    _properties: Option[List[GameProfileProperty]]
)
object PrivateUserData {
    def toStringBody(var_id: Object, var_properties: Object) =
        s"""
        | {
        | "id":$var_id,"properties":$var_properties
        | }
        """.stripMargin
}
